modules/apps/object/object-entry-folder-rest-impl/rest-openapi.yaml

```diff
@@ -1,8 +1,304 @@
+components:
+    schemas:
+        ObjectEntryFolder:
+            description:
+                Represents a object entry folder that contains objects entries and other object entry folders.
+            properties:
+                actions:
+                    additionalProperties:
+                        additionalProperties:
+                            type: string
+                        type: object
+                    # @review
+                    description:
+                        Block of actions allowed by the user making the request.
+                    readOnly: true
+                    type: object
+                assetLibraryId:
+                    # @review
+                    description:
+                        The ID of the asset library to which this object entry folder is scoped.
+                    format: int64
+                    type: integer
+                assetLibraryKey:
```

Comment:
We have remove the ALKey from the AssetLibraryObject schema in the new API

Reply:
My bad.

I will remove it, again :D